utils/db/helper.js

```diff
@@ -0,0 +1,9 @@
+import firebase from "../firebase/firebase";
+
+export async function getCurrentValue(url) {
```

Comment:
We should refactor all other database reads to use this function before this PR merges, that way there's a standard for when we want to do future database interaction

Reply:
I think I have done this already